packages/myst-to-react/src/crossReference.tsx

```diff
@@ -150,6 +153,7 @@ export function CrossReferenceHover({
     const el = document.getElementById(htmlId);
     scrollToElement(el, { htmlId });
   };
+  const isButtonLike = !!className?.includes('button');
```

Comment:
Do we need to do something that parses the class list a bit more carefully here?

Reply:
Yes, actually. I don't like this at all! 

(coffee brain)

Reply:
e.g. a string with `border shadow not-a-button` will turn on behaviour.

There is code for this in the admonitions I think.